sci-geosciences/josm-bin/josm-bin-18004.ebuild

```diff
@@ -0,0 +1,32 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit java-utils-2 desktop
+
+DESCRIPTION="Java-based editor for the OpenStreetMap project"
+HOMEPAGE="https://josm.openstreetmap.de/"
+SRC_URI="https://josm.openstreetmap.de/download/josm-snapshot-${PV}.jar"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND=">=virtual/jre-1.8"
+
+S="${WORKDIR}"
```

Comment:
Usually we put this right under SRC_URI with no blank line to match skel.ebuild ordering.